src/lib/homestore_backend/hs_pg_manager.cpp

```diff
@@ -60,6 +60,15 @@ PGManager::NullAsyncResult HSHomeObject::_create_pg(PGInfo&& pg_info, std::set<
     auto pg_id = pg_info.id;
     if (auto lg = std::shared_lock(_pg_lock); _pg_map.end() != _pg_map.find(pg_id)) return folly::Unit();
 
+    const auto most_avail_num_chunks = chunk_selector()->most_avail_num_chunks();
+    const auto chunk_size = chunk_selector()->get_chunk_size();
+    const auto needed_num_chunks = sisl::round_up(pg_info.size, chunk_size) / chunk_size;
```

Comment:
I think it`s better to use round_down here

Reply:
This ensures that the actual size occupied by the pg is smaller than the specified size, allowing pg moves on machines with sufficient remaining space. I agree with this excellent point.